hx_lti_initializer/views.py

```diff
@@ -213,6 +213,16 @@ def launch_lti(request):
         object_id = get_lti_value(settings.LTI_OBJECT_ID, tool_provider)
         course_id = str(course)
         if set(roles) & set(settings.ADMIN_ROLES):
+            try:
+                userfound = LTICourseAdmin.objects.get(
```

Comment:
@lduarte1991 The `username` should work fine for this actually, since the `username` will be the person's HUID.

I was thinking that maybe we should do email instead (i.e. `lis_person_contact_email_primary`), but the problem with that is people tend to have several harvard email addresses, and there's a chance the person entering the email might put in the wrong one. If we keep it as the `username` (HUID), then it's unambiguous. A little bit of a hassle to enter, but at least it will eliminate that potential for error. Maybe down the line, we can enhance it so that it can be either/or, but for now, let's stick to `username`.